### DISCUSSION TITLE
Wazuh Manager Disconnection Timeout Attacker

### DISCUSSION THREAD

**Discussion by BuffaloWill** [Security-Onion-Solutions/securityonion#9491] [Unsupported Versions] [answered]
It looks like by default a Wazuh agent is considered "Disconnected" from the manager if it hasn't phoned home in 30 minutes. Is it possible to set the agents_disconnection_time in Security Onion or otherwise decrease this value? I don't see that option in ossec.conf.

**Comment by dougburks** [comment, marked as answer]
It looks like the `agents_disconnection_time` option was added in Wazuh 4.10:
https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/global.html#agents-disconnection-time

So that option won't work since we use Wazuh 3.13:
https://docs.securityonion.net/en/2.3/wazuh.html#adding-agents
